cl/phase1/network/services/sync_committee_messages_service.go

```diff
@@ -52,6 +55,8 @@ func NewSyncCommitteeMessagesService(
 
 // ProcessMessage processes a sync committee message
 func (s *syncCommitteeMessagesService) ProcessMessage(ctx context.Context, subnet *uint64, msg *cltypes.SyncCommitteeMessage) error {
+	s.mu.Lock()
```

Comment:
do we need to protect the whole ProcessMessage here?

Reply:
because we can call it concurrently